lib/livebook_web/live/home_live.ex

```diff
@@ -154,6 +157,23 @@ defmodule LivebookWeb.HomeLive do
     end
   end
 
+  defp memory_notification(assigns) do
+    ~H"""
+    <%= if @app_service_url && @memory.free < 30_000_000 do %>
+      <div class="flex justify-between items-center border-b border-gray-200 pb-4 text-gray-700">
+        <span class="flex items-end">
+          <.remix_icon icon="alarm-warning-line" class="text-xl mr-2" />
+          <span>
+          Less than 30 MB of memory left, consider adding more resources to
+          <a class="font-semibold" href={@app_service_url} target="_blank">the instance</a>
+          or closing sessions below.
```

Comment:
```suggestion
          or closing running sessions.
```

And please include a link. :)